Baidak_Max/HW5/Task_3.py

```diff
@@ -0,0 +1,20 @@
+number = int(input("Enter a number: "))
```

Comment:
good job
look at this implementation:

number = int(input("Enter a number: "))
if number < 0:
    print("Enter a non-negative number.")
else:
    factorial = 1
    for i in range(1, number + 1):
        factorial *= i
    print(f"The factorial of {number} is: {factorial}")